### DISCUSSION TITLE
File Explorer Preview: Change Zoom level?

### DISCUSSION THREAD

**Discussion by dom6770** [sumatrapdfreader/sumatrapdf#3873] [Q&A]
When I enable the preview in the File Explorer and SumatraPDF is set as a handler, it always shows the PDF in a zoom level where it fits the whole page. This makes it impossible to recognize anything:

I've looked through the settings and tried out common zoom in/out hotkeys, but nothing works. Is there a way to change the zoom level?

**Comment by GitHubRulesOK** [comment]
so SumatraPDf has always been one page fits all formats but currently just 2 ?, 
so for zoom you can use the adobe one  (I do, when its working !!) However mine has NO Zoom just rotate and fit width.
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/5833656d-bc3b-48a0-8c86-9c6c5d5c1d6f)
